macOS/DuckDuckGo/Preferences/View/PreferencesAIChat.swift

```diff
@@ -68,6 +68,23 @@ extension Preferences {
                                           includeAppVersionParameter: true)
                         }
                     }
+
+                    if model.shouldShowOpenAIChatInSidebarToggle {
+                        ToggleMenuItem(UserText.aiChatOpenInSidebarToggle,
+                                       isOn: $model.openAIChatInSidebar)
+                        .accessibilityIdentifier("Preferences.AIChat.openInSidebarToggle")
+                        .onChange(of: model.openAIChatInSidebar) { newValue in
+                            if newValue {
+    //                            PixelKit.fire(AIChatPixel.aiChatSettingsOpenInSidebarTurnedOn,
```

Comment:
Placeholder for sending pixels. Will be added alongside all project scoped pixels.